profiles/models_test.py

```diff
@@ -0,0 +1,18 @@
+"""
+Model tests
+"""
+# pylint: disable=no-self-use
+from django.test import TestCase
+from profiles.factories import ProfileFactory, UserFactory
+
+
+class ProfileTests(TestCase):
+    """tests for the profile model"""
+
+    def student_id_on_save(self):
+        """test that a student id is set on save"""
+        user = UserFactory.create()
+        profile = ProfileFactory.create(user=user)
```

Comment:
actually, since the profile is created with a signal after the user is created, this call should just fail.